common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/EstsTelemetry.java

```diff
@@ -106,33 +132,28 @@ public void emit(@Nullable final Map<String, String> telemetry) {
      */
     public void emit(final String key, final String value) {
         final String correlationId = DiagnosticContext.getRequestContext().get(DiagnosticContext.CORRELATION_ID);
-        emit(correlationId, key, value);
+        final String compliantValueString = TelemetryUtils.getSchemaCompliantString(value);
+        emit(correlationId, key, compliantValueString);
     }
 
     private void emit(final String correlationId, final String key, final String value) {
-        RequestTelemetry currentTelemetryInstance = getCurrentTelemetryInstance(correlationId);
+        final CurrentRequestTelemetry currentTelemetryInstance = getCurrentTelemetryInstance(correlationId);
```

Comment:
nit: is this private function being used elsewhere?

Reply:
No it is not being elsewhere

Reply:
I can remove it and move the logic to the two param method

Reply:
Addressed